cmd/snap-confine/snap-confine.c

```diff
@@ -108,9 +260,17 @@ int main(int argc, char **argv)
 	// Use our super-defensive parser to figure out what we've been asked to do.
 	struct sc_error *err = NULL;
 	struct sc_args *args SC_CLEANUP(sc_cleanup_args) = NULL;
+	sc_preservation_context SC_CLEANUP(sc_cleanup_preservation_context)
+	    prsv_ctx = {
```

Comment:
Some weird formatting here. Did indent do this?

Reply:
Yes